travis/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -ex
 
-# Linux Debug
 ./sky/tools/gn --release
 ninja -j 4 -C out/Release
+./sky/tools/skyanalyzer --congratulate out/Release sky/sdk/example/stocks/lib/main.dart
```

Comment:
That's a good idea.  We should probably build it into the skyanalyzer (or some other) script so that it's easy for developers to run themselves.